portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ACSegmentsInDXP.testcase

```diff
@@ -206,15 +209,11 @@ definition {
 				category = "People",
 				portlet = "Segments");
 
-			AssertElementPresent(
-				key_tableEntry = "Dynamic Segment Test",
-				locator1 = "LexiconTable#TABLE_ENTRY_ELLIPSIS");
-		}
-
-		task ("Delete Dynamic Segment Test") {
-			Segmentation.deleteSegment(entry = "Dynamic Segment Test");
```

Comment:
Hey @CarlosBrichete , currently it is not possible to delete a segment that was created in AC by the DXP side, correct?

Reply:
I think you can delete the appearance of the segment in DXP, but the segment is not deleted from AC. In fact, the segment will appear again in DXP when the sync cycle time finishes.